src/Nethermind/Ethereum.HexPrefix.Test/HexPrefixTests.cs

```diff
@@ -36,8 +39,11 @@ public void Test(HexPrefixTest test)
 
         private class HexPrefixTestJson
         {
-            public byte[] Seq { get; set; }
+            [JsonPropertyName("seq")]
+            public int[] Seq { get; set; }
```

Comment:
Property is a list of numbers; deserialising a byte array wants to byte64 decode it

Reply:
Won't a custom converter fix it?

Reply:
Yeah but we have customer converters to output `0x...` for `byte[]`; is just a test type don't see reason to put too much effort in here rather than just change to `int[]`?